### PR TITLE
Switch to Preview1 pre-release label

### PR DESCRIPTION
**Pull request by gkhanna79**

### CODE REVIEW

**Comment by gkhanna79** [comment]
@eerhardt @ramarag PTAL

File: src/Microsoft.Extensions.DependencyModel/project.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "description": "Abstractions for reading `.deps` files.",
-  "version": "2.0.0-beta-*",
+  "version": "2.0.0-preview1-*",
```
Comment (eerhardt) [inline src/Microsoft.Extensions.DependencyModel/project.json]: Also, https://github.com/dotnet/core-setup/blob/master/src/Microsoft.DotNet.PlatformAbstractions/project.json#L3

Comment (gkhanna79) [inline src/Microsoft.Extensions.DependencyModel/project.json]: Fixed.

**Comment by gkhanna79** [comment]
Fixes #1605

**Pull request merged**